index.html

```diff
@@ -1109,8 +1109,9 @@ <h3>Data delivery</h3>
           Let |state:PressureState| be an [=adjusted pressure state=] given |data| and |source|.
         </li>
         <li>
-          Let |timestamp:DOMHighResTimeStamp| be a timestamp representing the time the |data| was
-          obtained from the |relevantGlobal|'s [=platform collector=].
+          Let |timestamp| be an platform-specific timestamp converted in an [=implementation-defined=]
+          way to an [=monotonic clock/unsafe current time=] using the same [=monotonic clock=]
+          that is shared by [=environment settings object/time origin=].
```

Comment:
I think it'd be helpful to add an `<aside>` here with something similar to https://github.com/w3c/sensors/blob/d8b0f67c3acc27330f97554185a32577a7d9d063/index.bs#L1429 so that it's clear that we do this to (if necessary) transform some platform-provided timestamp into an unsafe moment relative to the monotonic clock.

Reply:
Reopening, the suggested change above has not been incorporated.